include/nlohmann/json.hpp

```diff
@@ -3793,7 +3794,7 @@ class basic_json
             const auto it = find(key);
             if (it != end())
             {
-                return *it;
+                return static_cast<ValueType>(*it);
```

Comment:
Curious if there is a technical reason that we use `static_cast<ValueType>` here but `get<ValueType>` in other places.

Reply:
good catch